.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - 'main'
+  workflow_dispatch:
```

Comment:
I don't think we need this, do we? 🤔

Reply:
I initially added workflow_dispatch for easier testing in my fork, but you're right - it's not really needed in the main repo. I'll remove it!